tasks/main.yml

```diff
@@ -6,16 +6,21 @@
     - '{{ ansible_distribution }}_{{ ansible_distribution_major_version }}.yml'
     - '{{ ansible_os_family }}.yml'
   tags:
-    - 'role::$role'
-    - 'role::$role:install'
-    - 'role::$role:config'
+    - 'role::icinga2_client'
+    - 'role::icinga2_client:install'
+    - 'role::icinga2_client:config'
 
 - import_tasks: installation.yml
   tags:
-    - 'role::$role'
-    - 'role::$role:install'
+    - 'role::icinga2_client'
+    - 'role::icinga2_client:install'
 
 - import_tasks: configuration.yml
   tags:
-    - 'role::$role'
-    - 'role::$role:config'
+    - 'role::icinga2_client'
+    - 'role::icinga2_client:config'
+
+- import_tasks: certificates.yml
```

Comment:
Is this idempotent? I think you only need to get a new ticket if the agent packages have been installed and if there's no certificate in place yet?

Reply:
This is not yet finished and will be updated once the role is working properly